tests/operators/test_great_expectations.py

```diff
@@ -0,0 +1,154 @@
+"""
+Unit test module to test Great Expectations Operators.
+
+Requirements can be installed in, for instance, a virtual environment, with::
+
+    pip install -r requirements.txt`
+
+and tests can be run with, for instance::
+
+    pytest -p no:warnings
+"""
+
+from contextlib import contextmanager
+import logging
+import os
+from pathlib import Path
+import unittest.mock as mock
+
+from airflow.exceptions import AirflowException
+from great_expectations_provider.operators.great_expectations import (
+    GreatExpectationsOperator,
+)
+from great_expectations.exceptions.exceptions import GreatExpectationsError
+
+import pytest
+
+
+logger = logging.getLogger(__name__)
+
+# Set relative paths for Great Expectations directory and sample data
+base_path = Path(__file__).parents[2]
+good_data_file = os.path.join(
+    base_path, "include", "data/yellow_tripdata_sample_2019-01.csv"
+)
+bad_data_file = os.path.join(
+    base_path, "include", "data/yellow_tripdata_sample_2019-02.csv"
+)
+ge_root_dir = os.path.join(base_path, "include", "great_expectations")
+
+
+def test_great_expectations_operator__valid_expectation_suite():
+    operator = GreatExpectationsOperator(
+        task_id="task_id",
+        expectation_suite_name="taxi.demo",
+        batch_kwargs={"path": good_data_file, "datasource": "data__dir"},
+        data_context_root_dir=ge_root_dir,
+    )
+    result = operator.execute(context={})
+    logger.info(result)
+    assert result["success"] == True
+
+
+def test_great_expectations_operator__invalid_expectation_suite():
+    operator = GreatExpectationsOperator(
+        task_id="task_id",
+        expectation_suite_name="invalid-expectation-suite.name",
+        batch_kwargs={"path": good_data_file, "datasource": "data__dir"},
+        data_context_root_dir=ge_root_dir,
+    )
+    with pytest.raises(GreatExpectationsError):
+        operator.execute(context={})
+
+
+def test_great_expectations_operator__valid_checkpoint():
+    # This is ugly, but the older version of checkpoint configs that this supports
+    # doesn't seem to allow relative paths for the data assets being validated.
```

Comment:
Hi @jeffkpayne ! Could we modify this comment a bit? Maybe something like
```suggestion
    # Using this approach since older version of checkpoint configs that this supports
    # don't allow relative paths for the data assets being validated.
```

Reply:
(I think it's probably fine to remove the comment entirely - we do use a similar approach in the main Great Expectations codebase in situations like this)